mainwindow.cpp

```diff
@@ -278,19 +288,13 @@ MainWindow::~MainWindow()
 void MainWindow::on_pushButton_WordlistFileBrowse_clicked()
 {
     // We pop a dialog to choose a file to open.
-    // TODO: *.lst for file extension?
-    // TODO: Copy-pasting is evil! (open password file)
-    // TODO: What happens when John writes something while dialog
-    //       opened?
-    // TODO: Move dialog creation and setting up into window constructor.
     QFileDialog dialog;
     dialog.setFileMode(QFileDialog::ExistingFile);
-    // TODO: Dialog could allow user to select multiple files. May it
-    //       be good to support this ability? To concatenate selected file?
+    dialog.setNameFilter(tr("World list files (*.lst)"));
+
```

Comment:
On Thu, May 14, 2015 at 11:26:17PM -0700, Shinnok wrote:
> >      QFileDialog dialog;
> >      dialog.setFileMode(QFileDialog::ExistingFile);
> > -    // TODO: Dialog could allow user to select multiple files. May it
> > -    //       be good to support this ability? To concatenate selected file?
> > +    dialog.setNameFilter(tr("World list files (*.lst)"));
> > +
> 
> Keep the *.lst default, but add these two extra filters:
> * Dict files (*.dict)
> * Any files (*)
> 
> We don't want the user to not be able to load wordlists such as /usr/share/dict/words.

I guess *.txt files should be added too.

Thanks!

-- 
Regards,
Aleksey Cherepanov